prime-router/metadata/tables/zip-code-data.csv

```diff
@@ -19,6 +19,7 @@ state_fips,state,state_abbr,zipcode,county,city
 1,Alabama,AL,35043,Shelby,Chelsea
 1,Alabama,AL,35044,Talladega,Coosa pines
 1,Alabama,AL,35045,Chilton,Clanton
+1,Alabama,AL,35046,Chilton,Clanton
```

Comment:
I'm worried that our zip code data seems to be missing so many zip codes

Reply:
I agree.  We may want research 3rd party APIs that could be used to validates.  Note, I got a two month file from Compass Urgent Care on Friday, and it appears that I've got to research around 20 missing zip codes, so I'll be putting in another Zip Code update request today.